app/src/main/java/com/xinto/opencord/domain/activity/ActivityType.kt

```diff
@@ -0,0 +1,16 @@
+package com.xinto.opencord.domain.activity
+
+import com.github.materiiapps.enumutil.FromValue
+
+@FromValue
+enum class ActivityType(val value: Int) {
+    Game(0),
+    Streaming(1),
+    Listening(2),
+    Watching(3),
+    Custom(4),
+    Competing(5),
+    Unknown(-1);
```

Comment:
I think Unknown should be the first item since it has the least value out of everything else, but that's just my preference, feel free to argue

Reply:
I'm just refactoring how this is structured not reordering everything so a bit out of scope for this pr.
However, unknown is a temporary fix for the time being